Document API tokens on-prem 24.04 (#3229)
centreon/centreon-documentation: versioned_docs/version-24.04/api/api-tokens.md

---
id: api-tokens
title: API tokens
---

> Using the API, a third-party application can perform actions in Centreon. To be able do that, the third-party application must authenticate itself to Centreon, using a user account and an API token.
> 

Reviewer: ```suggestion
Using the API, a third-party application can perform actions (webhook) in Centreon using an API token.

```

A token is linked to one [Centreon user](../monitoring/basic-objects/contacts.md) and is valid only for a certain period of time. API calls will be executed according to the [rights assigned to this user](../administration/access-control-lists.md#granting-rights-to-a-user). A specific user can have several tokens.

## Who can create API tokens?

> * [Users with an **Editor** or **Administrator** role](../administration/access-control-lists.md#granting-rights-to-a-user) can access the **Administration > API Tokens** page and can create tokens for themselves.
> * Users with an **Administrator** role can also see tokens created by other users, can create tokens for other users, and can disable or delete them.

Reviewer: ```suggestion
* [Users or administrators](../administration/access-control-lists.md#granting-rights-to-a-user) accessing the **Administration > API Tokens** page can create tokens for themselves.
* Administrators can also see tokens created by other users, can create tokens for other users, and can disable or delete them.
```
